src/gui/dialogs/preferences_dialog.cpp

```diff
@@ -813,7 +813,9 @@ listbox& preferences_dialog::setup_hotkey_list(window& window)
 	hotkey_list.clear();
 	visible_hotkeys_.clear();
 
-	std::string text_feature_on =  "<span color='#0f0'>" + _("&#9679;") + "</span>";
+	std::string text_game_feature_on = "<span color='#0f0'>" + _("game_initial^G") + "</span>";
+	std::string text_editor_feature_on = "<span color='#0f0'>" + _("editor_initial^E") + "</span>";
+	std::string text_title_feature_on = "<span color='#0f0'>" + _("title_initial^T") + "</span>";
```

Comment:
Maybe add po comments (`// TRANSLATORS:`) that explain how to translate these. I don't think the `context^` markers alone are clear enough.

Reply:
I'm actually not sure where to add such comment - is it just the line before this block or should I apply to every line?

Reply:
Actually, I just noticed, I don't think even the G/E/T initials on the heading are translated and I can't see in the code how they are generated (I don't really know GUI2 that well). For these markers to be translated the headings should be translated too. For example, Game is Jeu in French yet the heading is still listed as G (although the tool-tip is translated).

Reply:
> I'm actually not sure where to add such comment - is it just the line before this block or should I apply to every line?

A `TRANSLATORS:` or `po:` comment must be on the line immediately preceding the translated string. See `--add-comments` in the xgettext manual: https://www.gnu.org/software/gettext/manual/html_node/xgettext-Invocation.html

I was told before to place such comments before every single translated string they apply to.

> Actually, I just noticed, I don't think even the G/E/T initials on the heading are translated and I can't see in the code how they are generated (I don't really know GUI2 that well).

Here:

```
$ grep -Rwn G data/gui/
data/gui/window/preferences/02_hotkeys.cfg:134:					label = _ "game_hotkeys^G"
```

And it _is_ already translated:

```
$ grep -A 1 game_hotkeys.G po/*/fr.po 
po/wesnoth-lib/fr.po:msgid "game_hotkeys^G"
po/wesnoth-lib/fr.po-msgstr "G"
```

I assume you should make the PR use the same string (`"game_hotkeys^G"`) on the C++ side, so as to reuse the existing translation.

For orientation, in general, the foo.cpp dialog uses a data file called foo.cfg. For example, src/gui/dialogs/statistics_dialog.cpp uses data/gui/window/statistics_dialog.cfg. The cpp file "finds" the cfg file using the id, in this case, `id = "statistics_dialog"`. The preferences dialog is a bit more complicated because it's tabbed.

Reply:
Thanks, I suspected it must have been in a cfg somewhere. It's clear that at least the French translators didn't know what to do with G/E/T, so maybe I should check what notes are in the cfg as well, if any.

I hope if I re-use the game_hotkeys^G, etc, then they only need to be translated once. I'll try to investigate this more after the weekend.

Reply:
There aren't many translations that know to change the G/E/T initials according to their language but Galego (Galician) appears to be one of them. I'm pleased to report that re-using the existing `game_hotkeys^G` string provides the translated initial for the markers as well. So I'll update this shortly.